build.gradle

```diff
@@ -1,10 +1,13 @@
 buildscript {
     ext {
-        compose_ui_version = '1.2.0'
+        compose_ui_version = "1.3.3"
+        hilt_version = "2.44.2"
```

Comment:
We could remove that from here later on. Tough to maintain. 

Later on we could create buildSrc module.

Reply:
@mateuszteteruk
Or use version catalogs:
https://twitter.com/ZacSweers/status/1623110975439224839